src/methods/simulate.jl

```diff
@@ -18,15 +18,16 @@ function simulate{T}(sys::LtiSystem{Val{T},Val{:cont}}, tspan;
 
   f     = (t,x,dx)->sys(t,x,dx,input)
   simvar= SimType(initial, zeros(numoutputs(sys)), zeros(numinputs(sys)))
+  f(tspan[1], simvar, zeros(initial))
 
   tstops= [tstops..., discontinuities(input, tspan)...]
   prob  = ODEProblem(f, simvar, tspan)
   sln   = OrdinaryDiffEq.solve(prob, alg; tstops = tstops, kwargs...)
 
   tvals = sln.t
-  xvals = Matrix{eltype(tvals)}(length(tvals), numstates(sys))
-  yvals = Matrix{eltype(tvals)}(length(tvals), numoutputs(sys))
-  uvals = Matrix{eltype(tvals)}(length(tvals), numinputs(sys))
+  xvals = Matrix{eltype(sln[1].x)}(length(tvals), numstates(sys))
+  yvals = Matrix{eltype(sln[1].y)}(length(tvals), numoutputs(sys))
+  uvals = Matrix{eltype(sln[1].u)}(length(tvals), numinputs(sys))
```

Comment:
This was needed for SysId, to be able to take the derivative of a cost function based on the simulation response of a parameterized LtiSystem.